apps/web/utils/actions/safe-action.ts

```diff
@@ -55,6 +55,9 @@ export const actionClient = baseClient
     // validate user owns this email
     const emailAccount = await prisma.emailAccount.findUnique({
       where: { id: emailAccountId },
+      include: {
```

Comment:
Use `select` here. Fetch only the data we need. Also more secure, especially important when fetching account data.